src/Helmet.js

```diff
@@ -134,6 +142,22 @@ const updateTitle = title => {
     document.title = title || document.title;
 };
 
+const updateHtmlAttributes = (attributes) => {
+    const htmlTag = document.getElementsByTagName("html")[0];
+
+    const oldAttributeCount = htmlTag.attributes.length;
+    if (oldAttributeCount) {
+        for (let i = oldAttributeCount - 1; i >= 0; i--) {
+            htmlTag.removeAttribute(htmlTag.attributes[i].name);
```

Comment:
How come we are removing the existing html attributes?  I thought we would be adding these attributes in addition.  Currently for the other tags we don't remove anything the developer already put in place.